queries/wgsl_bevy/highlights.scm

```diff
@@ -14,7 +14,7 @@
 
 (function_declaration
   (import_path
-    ((identifier) @function.)))
+    (identifier) @function.))
```

Comment:
Where's the space? :thinking:

Reply:
It was a remnant from #5854 that was not looked at